swagger-templates/src/main/resources/OktaJava/User.mustache

```diff
@@ -14,46 +14,14 @@
     limitations under the License.
 }}
 
-    /**
```

Comment:
We cannot continue support this anymore since the new spec (https://github.com/okta/openapi/blob/master/dist/spec.yaml#L5412) has replaced `addFactor(...)` with `enrollFactor(...)`. The new method `enrollFactor(...) has `UserFactor` object as return type. Therefore, commented these two methods from the templates.